cmake/toolchain/zephyr/0.10/target.cmake

```diff
@@ -7,9 +7,14 @@ else()
   set(CROSS_COMPILE_TARGET_x86 i586-zephyr-elf)
 endif()
 
+if(CONFIG_64BIT)
```

Comment:
We don't need this, we should be able to use the riscv64-zephyr-elf for both 32-bit and 64-bit builds